### PR TITLE
feat: color in new file and style improvement

### PR DESCRIPTION
**Pull request by KoTuK2306**

### CODE REVIEW

File: colors.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:root {
+  --primaryColor: #88888850;
```
Comment (pavelsokolov21) [inline colors.css]: what's color with too many characters?

File: index.html [hunk 2]
```diff
@@ -15,15 +15,18 @@ <h1 class="title">Calculator</h1>
...
+        <button class="button">&#9003</button>
         <button class="button">1/x</button>
-        <button class="button">x^2</button>
-        <button class="button">x^(1/2)</button>
-        <button class="button">/</button>
+        <button class="button">
+          x
+          <sup>2</sup>
+        </button>
+        <button class="button">&#8730x</button>
```
Comment (pavelsokolov21) [inline index.html]: `&#8730` and `x`?

File: index.html [hunk 3]
```diff
@@ -15,15 +15,18 @@ <h1 class="title">Calculator</h1>
...
+        <button class="button">÷</button>
         <button class="button">7</button>
         <button class="button">8</button>
         <button class="button">9</button>
-        <button class="button">x</button>
+        <button class="button">X</i></button>
```
Comment (pavelsokolov21) [inline index.html]: the closing tag `</i>` without the opened

File: styles.css [hunk 4]
```diff
@@ -42,4 +41,32 @@ body {
   padding: 0;
   font-family: inherit;
   font-size: 20px;
+  font-weight: bold;
+  color: inherit;
```
Comment (pavelsokolov21) [inline styles.css]: why `color: inherit`?

File: styles.css [hunk 5]
```diff
@@ -42,4 +41,32 @@ body {
...
+.button:nth-child(n + 21) {
+  border-bottom: 1px solid var(--secondaryColor);
+}
+
+.button:nth-child(24) {
+  background-color: var(--resultButtonColor);
+}
+
+.button:nth-child(-n + 8) {
+  background-color: var(--operButtonsColor);
```
Comment (pavelsokolov21) [inline styles.css]: does `oper` mean `operation`? Use full naming, please

File: styles.css [hunk 6]
```diff
@@ -42,4 +41,32 @@ body {
...
+  border-top: 1px solid var(--secondaryColor);
+  cursor: pointer;
+}
+
+.button:nth-child(4n) {
+  border-right: 1px solid var(--secondaryColor);
+  background-color: var(--operButtonsColor);
+}
+
+.button:nth-child(n + 21) {
```
Comment (pavelsokolov21) [inline styles.css]: `nth-child` with a weird expression looks unclear. Just create another class and add it to elements

File: styles.css [hunk 7]
```diff
@@ -42,4 +41,32 @@ body {
...
+  border-bottom: 1px solid var(--secondaryColor);
+}
+
+.button:nth-child(24) {
+  background-color: var(--resultButtonColor);
+}
+
+.button:nth-child(-n + 8) {
```
Comment (pavelsokolov21) [inline styles.css]: the same about weird expressions

File: styles.css [hunk 8]
```diff
@@ -42,4 +41,32 @@ body {
...
+.button:nth-child(4n) {
+  border-right: 1px solid var(--secondaryColor);
+  background-color: var(--operButtonsColor);
+}
+
+.button:nth-child(n + 21) {
+  border-bottom: 1px solid var(--secondaryColor);
+}
+
+.button:nth-child(24) {
```
Comment (pavelsokolov21) [inline styles.css]: I think it's `last-chid` or the situation to make another class

File: styles.css [hunk 9]
```diff
@@ -42,4 +41,32 @@ body {
   padding: 0;
   font-family: inherit;
   font-size: 20px;
+  font-weight: bold;
+  color: inherit;
+  border-radius: 0;
+  border: 0;
```
Comment (pavelsokolov21) [inline styles.css]: Is it necessary?

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: styles.css (hunk 9 above)
Comment (KoTuK2306) [reply styles.css]: Yep

File: colors.css [hunk 10]
```diff
@@ -0,0 +1,7 @@
+:root {
+  --primaryColor: rgba(136, 136, 136, 0.314);
+  --secondaryColor: #262626;
+  --operationButtonsColor: rgba(54, 78, 113, 0.337);
+  --resultButtonColor: rgba(54, 78, 113, 0.698);
```
Comment (pavelsokolov21) [inline colors.css]: don't u seriously need `0.002`? :)  I think 0.7 should be great too

File: colors.css [hunk 11]
```diff
@@ -0,0 +1,7 @@
+:root {
+  --primaryColor: rgba(136, 136, 136, 0.314);
+  --secondaryColor: #262626;
+  --operationButtonsColor: rgba(54, 78, 113, 0.337);
```
Comment (pavelsokolov21) [inline colors.css]: 0.3

File: colors.css [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+  --resultButtonColor: rgba(54, 78, 113, 0.698);
+  --hoverColor: rgba(179, 97, 255, 0.412);
```
Comment (pavelsokolov21) [inline colors.css]: 0.4

File: styles.css (hunk 4 above)
Comment (pavelsokolov21) [reply styles.css]: fix that

File: styles.css [hunk 13]
```diff
@@ -42,4 +41,31 @@ body {
...
+  border-left: 1px solid var(--secondaryColor);
+  border-top: 1px solid var(--secondaryColor);
+  cursor: pointer;
+}
+
+.operation {
+  background-color: var(--operationButtonsColor);
+}
+
+.rightBorder {
```
Comment (pavelsokolov21) [inline styles.css]: This is a bad name for class. You might add other properties to this class and this class won't match the name. Or use a modification from the BEM methodology (e.g. button_right)

File: styles.css [hunk 14]
```diff
@@ -42,4 +41,31 @@ body {
...
+  border-right: 1px solid var(--secondaryColor);
+}
+
+.bottomBorder {
```
Comment (pavelsokolov21) [inline styles.css]: the same about class naming

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: styles.css [hunk 15]
```diff
@@ -42,4 +41,31 @@ body {
...
+  border-left: 1px solid var(--secondaryColor);
+  border-top: 1px solid var(--secondaryColor);
+  cursor: pointer;
+}
+
+.operation {
+  background-color: var(--operationButtonsColor);
+}
+
+.border_right {
```
Comment (pavelsokolov21) [inline styles.css]: button_right

File: styles.css [hunk 16]
```diff
@@ -42,4 +41,31 @@ body {
...
+  border-right: 1px solid var(--secondaryColor);
+}
+
+.border_bottom {
```
Comment (pavelsokolov21) [inline styles.css]: button_bottom

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

**Review by pavelsokolov21 — APPROVED** [review]

**Pull request merged**